biopet-autodoc/synchronize_docs.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
```

Comment:
For some extra checking, you may want set `set -eu` and `set -o pipefail` in the beginning of the script. This makes sure bash ends when it encounters an error or an unset variable.